randomize

```diff
@@ -582,6 +583,11 @@ const yargs = require('yargs')
     describe: 'Enable tournament mode',
     type: 'boolean',
   })
+  .option('accessibility-patches', {
```

Comment:
Rename to `disable-accessibility-patches` to more accurately reflect what this option does.